bark/benchmark/benchmark_result.py

```diff
@@ -39,28 +39,35 @@ def configs_from_dict(behavior_dict):
 class BenchmarkConfig:
     def __init__(self, config_idx, behavior_config,
                  scenario, scenario_idx, scenario_set_name,
-                 scenario_set_param_desc=None):
+                 scenario_set_param_desc=None, track_id_ego=None, 
+                 track_file_name=None):
         self.config_idx = config_idx
         self.behavior_config = behavior_config
         self.scenario = scenario
         self.scenario_idx = scenario_idx
         self.scenario_set_name = scenario_set_name
         self.scenario_set_param_desc = scenario_set_param_desc or {}
+        self.track_id_ego = track_id_ego
```

Comment:
What is Data-scenario generation dependent should become part of the scenario set param description, which was introduced for the purpose of maintaining ScenarioGeneration specific additional infos.

Reply:
okay, I implemented it using scenario_set_param_desc